manifests/init.pp

```diff
@@ -117,12 +133,16 @@
 # @author Trevor Vaughan <tvaughan@onyxpoint.com>
 #
 class simp (
+  # This parameter set from data in modules
   Hash                            $scenario_map,
-  Enum['simp',
-       'simp_lite',
-       'poss',
-       'none',
-       'remote_access']           $scenario                   = 'simp',
+  Enum[
```

Comment:
It's almost like this was a set to a String originally on purpose, to, I don't know, allow for the scenarios to be configurable per site or per iso build or even per hostgroup, and also to not pigeonhole us in the future. Funny that.